README.md

```diff
@@ -23,6 +23,24 @@ def deps do
 end
 ```
 
+You'll also need to add `:artifact` to your list of applications, again in `mix.exs`, something like:
+
+```elixir
+def application do
+  [
+    applications: [
+    	:artifact
+    ]
+  ]
+end
+```
+
+It's also recommended to install goon (which ```artifact```/```porcelain``` uses) download the package for your system [here](https://github.com/alco/goon/releases) and unzip the contents somewhere in your path, for example:
```

Comment:
This needs punctuation.  How about something like:

"Since `artifact` relies on external processes for transformations, it is recommended that you install the [`goon`](https://github.com/alco/goon/releases) middleware.  To install, download the package for your system and unzip the contents somewhere in your $PATH:"